index.md

```diff
@@ -1,14 +1,28 @@
 ---
 layout: page
 full-width: true
+hero: true
 ---
 
 <div class="jumbotron jumbotron-fluid" style="background: rgb(0,0,0);
 background: linear-gradient(90deg, rgba(0,0,0,1) 0%, rgba(20,33,61,1) 0%, rgba(20,33,61,1) 40%, rgba(0,48,150,1) 100%, rgba(229,229,229,1) 100%);
 ￼">
   <div class="container">
-    <h1 class="display-4"><span style="color: #fca311; font-weight: bold">x</span><span style="color: white">DSL</span></h1>
-    <p class="lead" style="color:white">A Python-Native Compiler Design Toolkit</p>
+    <br>
+    <br>
+    <br>
+    <br>
```

Comment:
I would advise against using `br`, and instead using CSS to define margins (leads to cleaner code and is easier on the accessibility tools). You can take advantage of the `em` CSS unit, where `1em` corresponds to the font size.